sources/commands/Hydrate.ts

```diff
@@ -37,8 +37,8 @@ export class HydrateCommand extends Command<Context> {
 
     const {default: tar} = await import(/* webpackMode: 'eager' */ `tar`);
 
-    if (!existsSync(installFolder))
-      mkdirSync(installFolder, {recursive: true});
+    // Recreate the folder in case it was deleted somewhere else:
+    await mkdir(installFolder, {recursive: true});
```

Comment:
I don't know if you see my previous comment, this needs to be moved below: the `mkdir` call needs to happen just before `tar.x`, not `tar.t`.

Reply:
sorry, my mistake